var/spack/repos/builtin/packages/psblas/package.py

```diff
@@ -0,0 +1,211 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Psblas(AutotoolsPackage):
+    """PSBLAS: Parallel Sparse BLAS. A library of distributed sparse
+    linear algebra with support for GPU and Multithread acceleration.
+    Part of the PSCToolkit: Parallel Sparse Computation Toolkit."""
+
+    # Url for your package's homepage here.
+    homepage = "https://psctoolkit.github.io/"
+    git = "https://github.com/sfilippone/psblas3"
```

Comment:
Please add the extension (see https://spack.readthedocs.io/en/latest/packaging_guide.html#git).

```suggestion
    git = "https://github.com/sfilippone/psblas3.git"
```